resources/quality/biqu/base/base_0.3_PETG_super.inst.cfg

```diff
@@ -0,0 +1,16 @@
+[general]
+version = 4
+name = Super Quality
+definition = biqu_base
+
+[metadata]
+setting_version = 15
+type = quality
+quality_type = super
+material = generic_petg
+variant = 0.3mm Nozzle
+
+[values]
+speed_layer_0 = 15
+wall_thickness = =line_width*4
+#retraction_extra_prime_amount = 0.5
```

Comment:
Our config parser doesn't like / understand comments, so this needs to be removed.